igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/service/TravisService.groovy

```diff
@@ -69,6 +69,17 @@ class TravisService implements BuildService{
         accounts
     }
 
+    void setAccounts() {
+        this.accounts = getAccounts()
+    }
+
+    void syncRepos() {
+        Response response = travisClient.usersSync(getAccessToken())
+        if (response.status != 200) {
```

Comment:
200 is typically not the only success code. Usually 200 <= status < 300 are considered OK.